qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1 Send CV.feature

```diff
@@ -0,0 +1,218 @@
+Feature: 3.1 Send CV.feature
+
+Background:
+Given user opened the page with the form to send CV
+
+@Priority:Critical
+Scenario Outline: 3.1_01 Different cases entering data in fields "Name", "Email", attach CV, mark checkbox
+When user enters name <Name>
```

Comment:
You should always specify where to enter any info:
When user enters <name> into "Name" field